DataLayer/SongLibrary/DownloadSongs.sh

```diff
@@ -3,3 +3,11 @@
 wget https://s3-us-west-2.amazonaws.com/ucsolarteam.onboardmedia.songs/1-06+Animal.mp3
 wget https://s3-us-west-2.amazonaws.com/ucsolarteam.onboardmedia.songs/1-11+Armageddon+It.mp3
 wget https://s3-us-west-2.amazonaws.com/ucsolarteam.onboardmedia.songs/1-14+Hysteria.mp3
+sudo apt-get install eyeD3
```

Comment:
Also mentioned in person, but we shouldn't be installing any programs in this script. Add this command to the README of the repo